src/contexts/globalLayout/GlobalNavbar/LoggedInNav.tsx

```diff
@@ -54,7 +54,13 @@ function LoggedInNav({ queryRef }: Props) {
   // `viewer` exists. however, the logout action that dismounts client:root:viewer
   // causes this component to freak out before the parent realizes it shouldn't
   // be rendering this child... need to figure out best practices here
-  return query.viewer?.__typename === 'Viewer' && query.viewer.user?.username ? (
+  if (!query || query.viewer?.__typename !== 'Viewer') {
```

Comment:
I moved this check up here so that I could set the `username` const below more easily.
I confirmed that I could still sign out without issue (referring to comment on L53-57 above)